modules/cudf/src/data_frame.ts

```diff
@@ -71,16 +77,43 @@ type JoinProps<
 type CombinedGroupByProps<T extends TypeMap, R extends keyof T, IndexKey extends string> =
   GroupBySingleProps<T, R>|Partial<GroupByMultipleProps<T, R, IndexKey>>;
 
+const NumericTypes = [
+  new Bool8,
+  new Float32,
+  new Float64,
+  new Int8,
+  new Int16,
+  new Int32,
+  new Int64,
+  new Uint8,
+  new Uint16,
+  new Uint32,
+  new Uint64,
+];
+
+const IntegralTypes = [
+  new Int8,
+  new Int16,
+  new Int32,
+  new Int64,
+  new Uint8,
+  new Uint16,
+  new Uint32,
+  new Uint64,
+];
+
+const FloatTypes = [new Float32, new Float64];
```

Comment:
Do we maybe want to move these to `dtypes.ts` and export them?

Reply:
Yeah I think that'd be good.